lab_4/main.py

```diff
@@ -48,22 +47,19 @@ def get_word(self, word_id: int) -> str:
         if not isinstance(word_id, int):
             raise ValueError
 
-        if word_id not in self.storage.values():
-            raise KeyError
-
         for key, value in self.storage.items():
             if value == word_id:
                 return key
 
+        raise KeyError
```

Comment:
better not raise an error like that, we need to understand clearly why it's raised